src/main/java/seedu/address/model/task/completable/Todo.java

```diff
@@ -46,6 +46,11 @@ public boolean equals(Object other) {
                 && otherTodo.getIsDone().equals(getIsDone());
     }
 
+    @Override
+    public String getStringByDate() {
+        return null;
```

Comment:
Need to implement. Why return null?

Reply:
Completable utilises polymorphism. Alternatively, can have an attribute that is null if it's a todo.